fuzzers/adacc-pure-concolic/fuzzer.py

```diff
@@ -0,0 +1,53 @@
+import os
+import shutil
+import subprocess
+
+from fuzzers import utils
+from fuzzers.afl import fuzzer as afl_fuzzer
+
+
+def build():
+    # Set flags to ensure compilation with SymCC
+    os.environ['CC'] = "/symcc/build/symcc"
+    os.environ['CXX'] = "/symcc/build/sym++"
+    os.environ['CFLAGS'] = ""
+    os.environ['CXXFLAGS'] = ""
+    os.environ['SYMCC_REGULAR_LIBCXX'] = "1"
```

Comment:
Please add some comments about what SYMCC_REGULAR_LIBCXX and SYMCC_NO_SYMBOLIC_INPUT do.